src/encoded/static/components/navigation/components/SearchBar.js

```diff
@@ -276,9 +295,9 @@ const SelectItemTypeDropdownBtn = React.memo(function SelectItemTypeDropdownBtn(
         <div className="search-item-type-wrapper">
             <DropdownButton id="search-item-type-selector" size="sm" variant="outline-secondary" disabled={disabled}
                 onSelect={(eventKey, evt) => { onChangeSearchItemType(eventKey); }}
-                title={getDropdownTitleText()}>
+                title={getDropdownTitleText()} onToggle={(isOpen, event) => { onToggleSearchItemType(isOpen, event); }}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX props should not use arrow functions](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4833117)

Reply:
Just do `onToggle={onToggleSearchItemType}` here.
This isn't big deal since DropdownButton is relatively shallow, but creating new function inside the render method, means new function is passed down to DropDownButton each time, and thus DropdownButton re-renders each time that parent is re-rendered (failing prop equality checl for `nextProps.onToggle === this.props.onToggle`)

Reply:
Actually somewhat moot since not a lot of props, parent component itself is memoized and if it changes, contents change...  but still good practice to avoid creating (esp. not-returned-from-memoized-func) functions to pass down as props in render method.